extensions/wikia/PhalanxII/js/SpecialPhalanx.js

```diff
@@ -76,15 +78,55 @@ require(['jquery', 'mw', 'phalanx', 'BannerNotification'], function($, mw, phala
 				});
 		}).
 
-        // handle custom expire field
-        on('change', '#wpPhalanxExpire', function(ev) {
-            var customExpireField = $('#wpPhalanxExpireCustom'),
-                selectedOption = $(this).find(':selected');
-
-            if (selectedOption.data('is-custom')) {
-                customExpireField.show().focus();
-            } else {
-                customExpireField.hide();
-            }
-        });
+		// handle custom expire field
+		on('change', '#wpPhalanxExpire', function(ev) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "ev". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)